File: mo-system/mo-system-service/src/main/java/ink/ckx/mo/system/listener/MyAnalysisEventListener.java
package ink.ckx.mo.system.listener;

import com.alibaba.excel.event.AnalysisEventListener;

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/23
 */
public abstract class MyAnalysisEventListener<T> extends AnalysisEventListener<T> {

    private String msg;

    public abstract String getMsg();
}

File: mo-admin/mo-admin-service/src/main/kotlin/ink/ckx/mo/admin/listener/MyAnalysisEventListener.kt
package ink.ckx.mo.admin.listener

import com.alibaba.excel.event.AnalysisEventListener

/**
 * @author chenkaixin
 * @description
 * @since 2023/11/23
 */
abstract class MyAnalysisEventListener<T> : AnalysisEventListener<T>() {

    abstract val msg: StringBuilder
}